pkgs/by-name/od/odoo16/package.nix

```diff
@@ -4,34 +4,14 @@
 , rtlcss
 , wkhtmltopdf
 , nixosTests
+, odoo_version ? "16.0"
+, odoo_release ? "20240924"
```

Comment:
```suggestion
```
I don't think we should have those in inputs. The hashes are not updated so things don't work as expected and we generally have overrides for that.